### PR TITLE
Added cmdlet for uploading bundles

### PR DESCRIPTION
**Pull request by kannanbalu**
Cmdlet for uploading bundles offline onto SDDC manager.
1. Cmdlet Start-UploadVCFBundle added
2. sample spec bundleUploadspec.json
3. Start-UPloadVCFBundle.md

### CODE REVIEW

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Start-UploadVCFBundle
```
Comment (PowerVCF) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Function name is not consistent with the module. Suggest Start-VCFBundleUpload.

File: docs/functions/Bundles/Start-UPloadVCFBundle.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Synopsis
+The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
+
+### Syntax
+```
+Start-UploadVCFBundle -json <path to json file>
+```
+
+### Description
```
Comment (PowerVCF) [inline docs/functions/Bundles/Start-UPloadVCFBundle.md]: Suggest expanding the description to explain that you first need to SCP the files to SDDC Manager as this implies that the function is doing the upload

File: powerVCF.psm1 [hunk 3]
```diff
@@ -2119,6 +2119,53 @@ Function Get-VCFUpgradables {
...
+  This example invokes the upload of a bundle onto SDDC Manager
+#>
+
+  Param (
+    [Parameter (Mandatory=$true)]
+      [ValidateNotNullOrEmpty()]
+      [string]$json
+  )
+
+  $headers = @{"Accept" = "application/json"}
```
Comment (PowerVCF) [inline powerVCF.psm1]: Check with Gary as he has made a change around headers to reduce code

Comment (GaryJBlake) [reply powerVCF.psm1]: **Replace:**
$headers = @{"Accept" = "application/json"}
$headers.Add("Authorization", "Basic $base64AuthInfo")

**With the new function, using the following line:**
createHeader # Calls Function createHeader to set Accept & Authorization

File: powerVCF.psm1 [hunk 4]
```diff
@@ -2119,6 +2119,53 @@ Function Get-VCFUpgradables {
...
+  $headers.Add("Authorization", "Basic $base64AuthInfo")
+
+  if ($PsBoundParameters.ContainsKey("json")) {
```
Comment (PowerVCF) [inline powerVCF.psm1]: Do you need this if statement when the json file is mandatory?

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 1 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: Fixed

File: docs/functions/Bundles/Start-UPloadVCFBundle.md (hunk 2 above)
Comment (kannanbalu) [reply docs/functions/Bundles/Start-UPloadVCFBundle.md]: Fixed

File: powerVCF.psm1 (hunk 3 above)
Comment (kannanbalu) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 (hunk 4 above)
Comment (kannanbalu) [reply powerVCF.psm1]: Fixed

**Pull request merged**